functions/aep-126-enum-case-consistent.js

```diff
@@ -0,0 +1,123 @@
+/**
+ * Validates that all enum values in a field use consistent case formatting.
+ *
+ * Based on AEP-126 specification (https://aep.dev/126).
+ *
+ * AEP-126 states: "All enum values should use a consistent case format across
+ * an organization." This rule checks that all values within a single enum use
+ * the same case style, without enforcing a specific case format.
+ *
+ * @param {object} field - The field object containing the enum
+ * @param {object} _opts - Options (unused)
+ * @param {object} context - Spectral context containing the path
+ * @returns {Array<object>} Array of error objects, or empty array if valid
+ */
+module.exports = (field, _opts, context) => {
+  if (!field || typeof field !== 'object') {
+    return [];
+  }
+
+  // Only check string enums
+  if (field.type !== 'string') {
+    return [];
+  }
```

Comment:
This will might properties that are declared `type: [null, string]`, which is the way nullable properties are defined in OpenAPI 3.1 and later. Please add some tests for v3.1 with nullable fields and then make sure all the rules work correctly.